djangocms_references/monkeypatch/admin.py

```diff
@@ -2,21 +2,23 @@
 from django.template.loader import render_to_string
 from django.urls import reverse_lazy
 
-from djangocms_alias import admin
+from djangocms_alias import admin as AliasOriginalAdmin
+from djangocms_snippet import admin as SnippetOriginalAdmin
 
 
-def _get_references_link(self, obj, request):
-    alias_content_type = ContentType.objects.get(
-        app_label=obj.alias._meta.app_label,
-        model=obj.alias._meta.model_name,
-    )
+def generate_get_references_link(content_grouper):
+    def _get_references_link(self, obj, request):
+        content_type = ContentType.objects.get(
+            app_label=getattr(obj, content_grouper)._meta.app_label, model=getattr(obj, content_grouper)._meta.model_name,
```

Comment:
extract `getattr(obj, content_grouper)._meta` and create a variable, as below:
`obj_grouper_meta = getattr(obj, content_grouper)._meta`

and then use `obj_grouper_meta.app_label` and `obj_grouper_meta.model_name`